src/main/groovy/nebula/plugin/responsible/NebulaFacetPlugin.groovy

```diff
@@ -144,34 +140,20 @@ class NebulaFacetPlugin implements Plugin<Project> {
             Set<Object> compileClasspath = new LinkedHashSet<Object>()
             compileClasspath.add(sourceSet.compileClasspath)
             compileClasspath.add(parentSourceSet.output)
-            compileClasspath.addAll(extractAllOutputs(parentSourceSet.compileClasspath))
+            compileClasspath.add(parentSourceSet.compileClasspath)
```

Comment:
I believe this duplicates all jars that are common between `sourceSet.compileClasspath` and `parentSourceSet.compileClasspath`.

If you can avoid changing the SourceSet's view of the classpath, this works out better in other places.